integration-tests/src/main/java/org/apache/druid/cli/CustomNodeRoleCommandCreator.java

```diff
@@ -23,6 +23,7 @@
 
 public class CustomNodeRoleCommandCreator implements CliCommandCreator
 {
+  @SuppressWarnings({ "rawtypes", "unchecked" })
```

Comment:
Nit: Is this required as part of the uber PR ? Can the annotation be removed as part of this PR ?